src/web/app/components/question-types/question-edit-answer-form/mcq-question-edit-answer-form.component.ts

```diff
@@ -31,10 +31,19 @@ export class McqQuestionEditAnswerFormComponent
 
   isMcqOptionSelected: boolean[] = [];
 
+  isOtherTicked: boolean = false;
+
+  @Output()
+  cssRefresh = new EventEmitter<boolean>();
```

Comment:
Would be good to specify the type here
```suggestion
  cssRefresh: EventEmitter<boolean> = new EventEmitter<boolean>();
```